src/saving/Save.cs

```diff
@@ -349,50 +359,3 @@ private static byte[] ReadBytesEntry(TarInputStream tar, int length)
         return buffer;
     }
 }
-
-/// <summary>
-///   Info embedded in a save file
-/// </summary>
-public class SaveInformation
-{
-    public enum SaveType
-    {
-        /// <summary>
-        ///   Player initiated save
-        /// </summary>
-        Manual,
-
-        /// <summary>
-        ///   Automatic save
-        /// </summary>
-        AutoSave,
-
-        /// <summary>
-        ///   Quick save, separate from manual to make it easier to keep a fixed number of quick saves
-        /// </summary>
-        QuickSave,
-    }
-
-    /// <summary>
-    ///   Version of the game the save was made with, used to detect incompatible versions
-    /// </summary>
-    public string ThriveVersion { get; set; } = Constants.Version;
-
-    public string Platform { get; set; } = FeatureInformation.GetOS();
-
-    public string Creator { get; set; } = Environment.UserName;
-
-    public DateTime CreatedAt { get; set; } = DateTime.Now;
-
-    /// <summary>
-    ///   An extended description for this save
-    /// </summary>
-    public string Description { get; set; } = string.Empty;
-
-    /// <summary>
-    ///   Unique ID of this save
-    /// </summary>
-    public Guid ID { get; set; } = Guid.NewGuid();
-
-    public SaveType Type { get; set; } = SaveType.Manual;
-}
```

Comment:
I moved this class to a new file, because I derived from this class.

Reply:
It's fine to be in a separate file, but I think deriving an invalid type from the class is not what I would have done. I would have made an additional constructor to create save info filled in with the invalid values.